test/libsolidity/gasTests/abiv2_optimised.sol

```diff
@@ -17,9 +17,9 @@ contract C {
 // optimize-yul: true
 // ----
 // creation:
-//   codeDepositCost: 626600
-//   executionCost: 657
-//   totalCost: 627257
+//   codeDepositCost: 664600
```

Comment:
Quite a difference... does this include the additional debug reason string?

Reply:
I checked, it includes 10 new code snippets. The whole contract is essentially only an ABI decoder, so i think this is reasonable.